.github/workflows/deploy.yml

```diff
@@ -27,6 +27,16 @@ jobs:
           cache: yarn
           cache-dependency-path: website/yarn.lock
 
+      - name: Cache code outputs
+        id: cache-code-outputs
+        uses: actions/cache@v3
+        env:
```

Comment:
what is this for?

Reply:
This is based on the example in the [documentation](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows)